docs/usage.md

```diff
@@ -62,10 +62,61 @@ the cloned repository.
 evg-module-manager disable --module wiredtiger
 ```
 
+## Handling git branches
+
+Use `git-branch` command to work on branch related cases on base repo and any enabled modules.
+By default, a git checkout action will be performed to the revision
+in the base repository. However, the `git-branch --operation` option can be provided to change this
+behavior.
+
+The option takes one of the following as an argument:
+
+* **checkout** [default] - Perform a `git checkout` to checkout the found revision.
+* **rebase** - Perform a `git rebase` to rebase changes on top of the found revision.
+* **merge** - Perform a `git merge` to merge changes up to the found revision into the current branch.
+
+### Checkout branches
+To create a new branch or checkout to a certain revision. The revision to be checked out would be the
+revision for the module in the evergreen manifest associated with the latest commit in the history that
+was run in evergreen.
+
+For the **checkout** option, you can specify a branch name to create with the `-b` or
+`--branch` option.
+
+#### Examples
+To create a branch named `my-branch`, use the following:
+
+```bash
+evg-module-manager git-branch -o checkout --branch my-branch
```

Comment:
I think it would be good to use long options in the documentation since they do a better job of describing what the option means. So here using `--operation` instead of `-o`.

Reply:
No, please do not use shortcut options in the documentation. The `--help` option can be used to determine the short options. The long options names provide better context as to what they do and switching between the different option types in the documentation is confusing.